api/src/routesUsers.js

```diff
@@ -72,7 +74,7 @@ async function create(req, res, next) {
       const result = await db
         .query(
           `INSERT INTO users (${keys}) VALUES ($1, $2, $3, $4, $5) RETURNING *`,
-          [first_name, last_name, email, salt, hash]
+          [first_name, last_name, email, Buffer.from(salt, 'hex'), hash]
```

Comment:
What does the Buffer do?

Reply:
The buffer creates a buffer object that converts the salt to binary from hexadecimal representation. After all my tinkering, this was the only way I could get it to function properly.